sf-stream/src/tests/test_context.rs

```diff
@@ -0,0 +1,374 @@
+// Copyright (c) Aptos
```

Comment:
Why are we duplicating this, this looks identical to the one we already have?

Reply:
There are some minor differences. We can add a todo to merge them, but we'd like to get this unblocked for the next stage.